views/js/controller/creator/creator.js

```diff
@@ -226,9 +229,17 @@ define([
                         if(isTestContainsItems()) {
                             const saveUrl = options.routes.save;
                             const testUri = saveUrl.slice(saveUrl.indexOf('uri=') + 4);
-                            return previewerFactory.init(decodeURIComponent(testUri), {
-                                readOnly: false,
-                                fullPage: true
+                            return previewerFactory(
+                                'qtiTest',
```

Comment:
Is it not possible to have this name managed trough a config instead?

Reply:
I am not sure that this should be in configs. 
https://github.com/oat-sa/extension-tao-item/blob/1b3fdb935f0298fc7dfd66d0d023c319199b7bfa/views/js/controller/items/action.js#L29 here also hardcoded
At what configs it should be put?